examples/query-expressions/query_expressions.bal

```diff
@@ -1,20 +1,33 @@
 import ballerina/io;
-
+ 
 public function main() {
     int[] nums = [1, 2, 3, 4];
-
-    // This is based on the `{ 10 × i | i ∈ nums }` set builder notation.
+ 
     // The `from` clause works similar to a `foreach` statement.
     int[] numsTimes10 = from var i in nums
                         // The `select` clause is evaluated for each iteration.
                         select i * 10;
-
     io:println(numsTimes10);
-
-    // This is based on the `{ i | i mod 2 = 0, i ∈ nums }` set builder notation.
-    int[] evenNums = from var i in nums
+ 
+    // Where clause can be used to filter iterable values.
+    // It can occur multiple times anywhere between the from and the select clause.
+    // This will pass the frame to the select clause only if `i % 2 == 0` is true.
```

Comment:
```suggestion
    // The `where` clause can be used to filter iterable values.
    // It can occur multiple times anywhere between the `from` and the `select` clause.
    // This will pass the frame to the `select` clause only if `i % 2 == 0` is true.
```